src/utils/conversion.ts

```diff
@@ -1219,6 +1219,15 @@ export function meshPermissionsToPermissions(
   };
 }
 
+/**
+ * @hidden
+ */
+export function bigNumberToU16(value: BigNumber, context: Context): u16 {
+  assertIsInteger(value);
+  assertIsPositive(value);
```

Comment:
What happens in the overflow case?

Does this need to assert its less than 2^16?

Reply:
Not sure, all others were also left as is